webroot/css/cake-core.css

```diff
@@ -0,0 +1,3 @@
+.cake-core-hidden {
```

Comment:
Well, as many don't use or rather need to use the cake.css or bake.css file, they definitely need the cake-core.css file so that postLink and maybe other core methods still work. Therefore I would serve it as a separated file.
My opinion. :wink: